benchmarks/cpp/lstm_cell.cpp

```diff
@@ -27,11 +27,11 @@ static void setupFusion(Fusion* fusion) {
 
   TensorView* tvs[16];
   for (size_t i = 0; i < 16; i++) {
-    tvs[i] = makeContigTensor(2, DataType::Float);
+    tvs[i] = makeSymbolicTensor(2, DataType::Float);
```

Comment:
this doesn't feel right. Looking at the chunk function, I think they are supposed to be contiguous in the fastest dimension.

Reply:
Correct. Changed to `setContiguity({false, true})` instead.